dist.ini

```diff
@@ -9,3 +9,14 @@ no_makemaker = 1
 alien_repo = http://www.mpg123.de/download
 alien_pattern_suffix = \.tar\.bz2
 alien_bins = mpg123
+
+[Prereqs / ConfigureRequires]
+Alien::Base = 0.021
+Alien::Base::ModuleBuild = 0.023
+Test::LoadAllModules = 0
+
+[Prereqs / Requires]
+Alien::Base = 0.021
+Alien::Base::ModuleBuild = 0.023
+Test::LoadAllModules = 0
```

Comment:
Looks like dzil already pulls in `Test::LoadAllModules`
https://metacpan.org/source/GETTY/Alien-mpg123-0.002/META.json#L38

Reply:
It is in CPAN, but my tests with latest Dist::Zilla do not put it in META files. Maybe the cpan tarball is not generated with the exact same dist.ini than this github repo ?
I updated to move Test::LoadAllModules in a "[Prereqs / TestRequires]"  section